test/OrchardCore.Tests/Utilities/ModelValidator.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace OrchardCore.Tests.Utilities
+{
+    public class ModelValidator
```

Comment:
This isn't used anywhere.

Reply:
This should be removed, it was used previously